doc/README.hbs

```diff
@@ -42,7 +42,7 @@ It accepts the following params:
 
 **Example**
 ```js
-const ResinAuth = require('resin-auth');
+import ResinAuth from 'resin-auth';
```

Comment:
This is the template for the readme, but the readme itself needs regenerating too. That should happen in build really, and get caught in CI, I'll take a look at why this didn't work. In the meantime, if you run `npm run readme` and then commit the changes we should be all good.

Reply:
The code above is TypeScript and the docs should be in JavaScript.

Reply:
Hmm, interesting. Should they?

If we're saying we should use ES6 patterns for internal modules, because it'll work nicely with TS and all usage will (eventually) be TS, should we not also write the docs for TS? If we don't this example code is pretty messy (`const ResinAuth = require('resin-auth').default`) and isn't likely to match anybody's usage of this module, since it's for internal use, which kind of defeats the point of having an example. I do like avoiding assuming too much about usage though...

Oh, wait, in fact: this code _is_ valid JavaScript, it's just it's ES6 module javascript. It'll work out of the box in Chrome, with FF/Edge/Node with a flag (presumably by default imminently), and for anybody using Babel/TS/etc. I'm pretty confident about assuming that. Thoughts?

Reply:
I'll let you decide that. I mentioned because `resin-token` and `resin-request` are like that. One is made in CS the other in TS but the README examples are in JS

Reply:
And even if we do, the README currently has ` ```js `. If we do show it in TS we need to change that bit

Reply:
> And even if we do, the README currently has ```js. If we do show it in TS we need to change that bit

Except this is valid vanilla JS. If we can say that we consider ES6 modules as standard JS now (which I think we effectively do internally, which is why all our internal modules now all follow ES6 module conventions), then we're accurate in just leave this marked as JS. I don't really care though, let's just do whatever's easiest and keep this moving @moranf

Reply:
Yes, whatever you think it's best :+1: